packages/html/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@interactors/html",
+  "version": "0.31.2",
+  "description": "Interactors for working with applications",
+  "main": "dist/cjs/index.js",
+  "browser": "dist/esm/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/thefrontside/interactors.git",
+  "author": "Frontside Engineering <engineering@frontside.com>",
+  "license": "MIT",
+  "files": [
+    "dist/**/*",
+    "src/**/*",
+    "README.md"
+  ],
+  "scripts": {
+    "clean": "rm -rf dist *.tsbuildinfo",
+    "lint": "eslint \"{src,test}/**/*.ts\"",
+    "mocha": "mocha -r ts-node/register",
+    "test:unit": "mocha -r ts-node/register \"test/**/*.test.ts\"",
+    "test:types": "dtslint types --localTs ../../node_modules/typescript/lib --expectOnly",
+    "test": "yarn test:unit && yarn test:types",
+    "prepack": "tsc --build && tsc --outdir dist/esm --module es2015 && tsc --outdir dist/cjs --module commonjs"
+  },
+  "dependencies": {
+    "@bigtest/globals": "^0.7.6",
+    "@bigtest/performance": "^0.5.0",
+    "change-case": "^4.1.1",
+    "element-is-visible": "^1.0.0",
+    "lodash.isequal": "^4.5.0"
+  },
+  "devDependencies": {
+    "@frontside/eslint-config": "^2.0.0",
+    "@frontside/tsconfig": "^1.2.0",
+    "@frontside/typescript": "^1.1.1",
+    "@types/jsdom": "^16.2.3",
+    "@types/lodash.isequal": "^4.5.5",
+    "@types/mocha": "^7.0.1",
+    "@types/node": "^13.13.4",
+    "dtslint": "^4.0.5",
+    "expect": "^24.9.0",
+    "express": "^4.17.1",
+    "jsdom": "^16.2.2",
+    "mocha": "^6.2.2",
+    "ts-node": "^9.1.1"
+  },
+  "volta": {
+    "node": "12.16.0",
+    "yarn": "1.19.1"
```

Comment:
We should move this repo to use npm

Reply:
https://github.com/thefrontside/interactors/issues/4